### PR TITLE
Add color accent to project cards

### PR DESCRIPTION
**Pull request by moteanup24**
Added colored bottom border per project protocol tag

Standardized project card length such that "donate" and "view details" buttons are aligned across cards

@dergigi

### CODE REVIEW

**Comment by moteanup24** [comment]
@dergigi pending your review

This PR implements the logic I laid out earlier: 
- if (tags contains 'nostr') then purple border
- if (tags contains 'lightning') then yellow border
- if (tags contains 'bitcoin') then orange border

I think this is more flexible than the tags[0] implementation but let me know if you prefer that and I can push that. If we take the tags[0] approach we would just need to monitor the projects mdx files to ensure that we have what we want in tags[0] of course (e.g. Zeus would probably need editing to place Lightning first, and we'd have to choose whether we want to highlight Bitcoin or Lightning in projects like graphene, which currently has both).

**Comment by moteanup24** [comment]
*Actually the current logic is 

if (tags contains 'nostr') then purple border
else if (tags contains 'lightning') then yellow border
else if (tags contains 'bitcoin') then orange border

**Comment by dergigi** [comment]
`else if` logic as described makes sense 👍  - will review shortly.

**Comment by dergigi** [comment]
I really like that! Well done! ✨ 

I think it will look even better if we remove the individual "Donate" buttons from the cards. I always thought they are misplaced there anyway. The card should get you to the project page (clicking anywhere, basically), which should have a prominent donate button. But that is a story for another PR.

File: components/ProjectCard.tsx [hunk 1]
```diff
@@ -22,10 +21,26 @@ const ProjectCard: React.FC<ProjectCardProps> = ({
...
+      'h-full space-y-4 rounded-xl border-b-4 border-purple-600 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
+  } else if (tags.includes('Lightning')) {
+    cardStyle =
+      'h-full space-y-4 rounded-xl border-b-4 border-yellow-300 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
+  } else if (tags.includes('Bitcoin')) {
+    cardStyle =
+      'h-full space-y-4 rounded-xl border-b-4 border-orange-400 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
+  } else {
+    cardStyle =
+      'h-full space-y-4 rounded-xl border-b-4 border-stone-100 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
```
Comment (dergigi) [inline components/ProjectCard.tsx:39]: NIT: lots of repeated code, the only thing that changes is the color.

**Review by dergigi — APPROVED** [review]

File: components/ProjectCard.tsx (hunk 1 above)
Comment (moteanup24) [reply components/ProjectCard.tsx:39]: Yes I noticed that (obviously, as I wrote it). Tailwind seems not to like dynamic colors, at least on my machine. Will look into how to clean this up, perhaps as part of a larger refactoring effort.

**Pull request merged**